### PR TITLE
MG-3355 feat: add GitHub actions to publish dd-etl to docker registry

### PR DESCRIPTION
**Pull request by ikanashov**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by mebelousov** [comment]
Зачем нужен отдельный Dockerfile?

**Comment by ikanashov** [comment]
> 

Хотелось чтобы каталоги dags и common были сразу внутри контейнера

**Comment by mebelousov** [comment]
> Хотелось чтобы каталоги dags и common были сразу внутри контейнера

Кажется, что это можно сделать, добавив новый таргет в Dockerfile.

File: data-detective-etl/Dockerfile [hunk 1]
```diff
@@ -66,3 +66,29 @@ VOLUME $AIRFLOW_HOME/dags \
...
+ && sed -i 's/\#PermitUserEnvironment no/PermitUserEnvironment yes/' /etc/ssh/sshd_config \
+ && mkdir $AIRFLOW_HOME/.ssh \
+ && echo "AIRFLOW_HOME=$AIRFLOW_HOME" > $AIRFLOW_HOME/.ssh/environment \
+ && echo "PYTHONPATH=$PATH:$AIRFLOW_HOME:$AIRFLOW_HOME/dags" >> $AIRFLOW_HOME/.ssh/environment \
+ && chown -R airflow:airflow $AIRFLOW_HOME
+EXPOSE 22
+
+USER airflow
+
+RUN mkdir $AIRFLOW_HOME/dags \
```
Comment (mebelousov) [inline data-detective-etl/Dockerfile:90]: Не нужно создавать папки.

https://docs.docker.com/engine/reference/builder/#copy
If <dest> doesn’t exist, it is created along with all missing directories in its path.

**Review by mebelousov — APPROVED** [review]
Отлично!

**Pull request merged**